scripts/jenkins/ardana/ansible/roles/reboot_node/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+#
+# (c) Copyright 2019 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+---
+- name: Fail when no reboot_target specified
+  fail:
+    msg: "{{ reboot_target }} not in {{ reboot_target_list }}"
+  when: reboot_target not in reboot_target_list
+
+# reboot deployer/admin node
+- include_tasks: reboot_deployer.yml
+  when: reboot_target == 'deployer'
+
+# reboot nodes - CROWBAR
+- include_tasks: reboot_cloud_nodes_crowbar.yml
```

Comment:
You can use the value of `cloud_product` here and remove the conditional `cloud_product == 'crowbar'`. That way when implementing the same for ardana you can just have a file named `reboot_cloud_nodes_ardana.yml` and will not need to change anything here.